log-generator/log-generator.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+process.env.NEW_RELIC_NO_CONFIG_FILE = true
+process.env.NEW_RELIC_APP_NAME = 'log-generator'
+process.env.NEW_RELIC_LICENSE_KEY = 'fill-in-license-key'
+
+const newrelic = require('newrelic')
+
+const yargs = require('yargs')
+const { hideBin } = require('yargs/helpers')
+const faker = require('faker')
+
+function getArgs() {
+  return yargs(hideBin(process.argv))
+    .scriptName('log-generator')
+    .usage(
+      'Usage: $0 OPTIONS\n\nstarts a process that will generate logs with either winston or pino'
+    )
+    .option('logtype', {
+      alias: 'l',
+      describe: 'which logger to use',
+      choices: ['winston', 'pino'],
```

Comment:
i think we should default all of these so we can just run `node log-generator.js`